Example/CryptoOnramp Example/CryptoOnramp Example/UI Helpers/PaymentMethodCardView.swift

```diff
@@ -46,7 +46,6 @@ struct PaymentMethodCardView: View {
             }
         }
         .aspectRatio(1.6 / 1, contentMode: .fit)
-        .frame(maxWidth: 280)
```

Comment:
It's not an extremely important detail, but this `maxWidth` was meant to keep the card from becoming giant in larger contexts. Feel free to adjust if you’d like, a `maxHeight` would probably also do the trick.

I’m also seeing we'll want to prevent the split view layout on iPad, but we can tackle that separately as that's unrelated here.

| without `maxWidth` | with `maxWidth` |
| --- | --- |
| <img src="https://github.com/user-attachments/assets/568750a2-5eec-4397-8c82-393501f40cdd" width="300" /> | <img src="https://github.com/user-attachments/assets/e6df7987-bff2-45bb-a840-c5ace5add6f6" width="300" /> |